apispec/openapi-circe/src/main/scala/sttp/tapir/openapi/circe/TapirOpenAPICirceToYaml.scala

```diff
@@ -0,0 +1,11 @@
+package sttp.tapir.openapi.circe
+
+import io.circe.syntax.EncoderOps
+import io.circe.yaml.Printer
+import sttp.tapir.openapi.OpenAPI
+
+trait TapirOpenAPICirceToYaml {
+  implicit class RichOpenAPI(openAPI: OpenAPI) {
+    def convertToYaml: String = Printer(dropNullKeys = true, preserveOrder = true).pretty(openAPI.asJson)
```

Comment:
isn't it the same functionality as `.toYaml`? Why would this be in the main sources?